app/src/sharedTest/java/org/oppia/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -147,6 +151,26 @@ class NavigationDrawerTestActivityTest {
     }
   }
 
+  @Test
+  fun testNavigationDrawerTestActivity_openNavigationDrawer_selectSwitchProfileMenu_showsExitToProfileChooserDialog() {
+    launch(NavigationDrawerTestActivity::class.java).use {
+      onView(withId(R.id.home_activity_drawer_layout)).perform(open())
+      onView(withText(R.string.menu_switch_profile)).perform(click())
+      onView(withText(R.string.home_activity_back_dialog_message)).check(matches(ViewMatchers.isDisplayed()))
```

Comment:
Update: For better code clarity write it like this:
`onView(withText(R.string.home_activity_back_dialog_message)).check(matches(isDisplayed()))`

Ditto in below test case.

Reply:
Also, after this change make sure to check that you do not have any unused imports.